### DISCUSSION TITLE
Add noWrap option to <Td> and <Th>

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#2405] [Ideas] [answered]
I would like to be able to control that some entries do not wrap (e.g. dates). But I am okay with some elements wrapping. Please allow setting this individually for `<Td>` and `<Th>`.

**Comment by TheSisb** [comment]
Hi @thomaschaaf,

This is a solid suggestion, we should add a `whiteSpace` style prop to those components!

We'll try to line that work up in the coming weeks, as our [current focus is on resiliency](https://github.com/twilio-labs/paste/discussions/2370) for the near term.

**Reply by TheSisb** [reply, marked as answer]
This feature has been released in `@twilio-paste/core` version `20.8.0`
